src/dataregistry/registrar_util.py

```diff
@@ -207,3 +208,37 @@ def _name_from_relpath(relative_path):
         name = base
 
     return name
+
+
+def _read_configuration_file(configuration_file, max_config_length):
+    """
+    Read a text, YAML, TOML, etc, configuration file.
+
+    Parameters
+    ----------
+    configuration_file : str
+        Path to configuration file
+    max_config_length : int
+        Maximum number of rows to read. Files above this length will be
+        truncated.
+
+    Returns
+    -------
+    contents : str
+    """
+
+    # Make sure file exists
+    if not os.path.isfile(configuration_file):
+        raise FileNotFoundError(f"{configuration_file} not found")
+
+    # Open configuration file and read up to max_config_length lines
```

Comment:
As noted in the comment on the docstring above, the argument refers to characters, not lines.   See e.g. https://docs.python.org/3.9/library/io.html#io.TextIOBase
I think characters is what we want.   The reason for the limit is to make sure some entry doesn't take up an unreasonable amount of space in the db.